app/subscriber/src/features/my-reports/edit/settings/template/ReportSchedule.tsx

```diff
@@ -44,8 +43,9 @@ export const ReportSchedule: React.FC<IReportScheduleProps> = ({ index, label })
           </p>
         </Col>
       </Col>
-      <Col className="frm-in">
+      <Col className="frm-in run-on">
         <label>Run On</label>
+        <div role="alert">{getIn(errors, `events.${index}.runOnWeekDays`)}</div>
```

Comment:
Placing the error here, rather than having it appear 7 times under each checkbox.